sort/countingSort.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import "fmt"
+
+//use this for strings
+func countingSortString(data []rune, size int) string {
```

Comment:
Remove this function as it is fairly understandable. Only work on simple int data type. Please a take at other sorting algorithms and make your code consistent with them.